packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-configs.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { POO_STATUSES } from './constants/constants';
+import VerificationStatusActionButton from 'Components/verification-status-action-button';
+
+export const getPOOStatusMessages = (status: typeof POO_STATUSES[keyof typeof POO_STATUSES]) => {
+    const resubmitButton = (onClick?: () => void) => (
+        <VerificationStatusActionButton button_text={<Localize i18n_default_text='Try again' />} onClick={onClick} />
+    );
+
+    const titles: Record<typeof status, React.ReactElement> = {
```

Comment:
For Localize to work properly, Shouldnt this be a function?

Reply:
Since it is a const wrapped in a function, the Localize will re-trigger